packages/@aws-cdk/cloudformation-include/README.md

```diff
@@ -135,7 +135,7 @@ All items unchecked below are currently not supported.
 - [x] Resources
 - [x] Parameters
 - [x] Conditions
-- [ ] Outputs
+- [x] Outputs
```

Comment:
Also add a section in the ReadMe about Outputs, similarly like the one that's already there for Conditions.